openshift/dev_notes.md

```diff
@@ -1,4 +1,12 @@
 ## Notes for developers
 
 ### Information on the downstreaming process can be found here:
-[https://github.com/openshift/operator-framework-tooling/blob/main/README.md](https://github.com/openshift/operator-framework-tooling/blob/main/README.md)
\ No newline at end of file
+[https://github.com/openshift/operator-framework-tooling/blob/main/README.md](https://github.com/openshift/operator-framework-tooling/blob/main/README.md)
+
+### (For Mac Os)
+
+To run the commands locally ensure that you have installed:
+
+```shell
+brew install findutils gnu-sed yq
```

Comment:
You actually don't need gnu-sed (i.e. `gsed`), the built in macOS `sed` works fine with the change you make.